code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -180,13 +180,36 @@
 	amount_per_transfer_from_this = 10
 	list_reagents = list(/datum/reagent/medicine/stimpak = 10, /datum/reagent/medicine/stimulants = 10, /datum/reagent/medicine/omnizine = 10)
 
+/obj/item/reagent_containers/hypospray/medipen/stimpak/Initialize()
```

Comment:
Unnecessary. `on_reagent_change()`, called in `Initialize()`, will call `update_icon()` anyway. Just remove this entire override.

Reply:
Done, deleted and gone.